packages/oid4vci-demo-frontend/src/router/AppRouter.tsx

```diff
@@ -23,6 +24,7 @@ export const routes: Record<string, any> = {
     '/information/success': <SSIInformationSuccessPage/>,
     '/credentials/select': <SSISelectCredentialPage/>,
     '/credentials/verify/request': <SSICredentialVerifyRequestPage/>,
+    '/credentials/info/vp/request': <SSIInformationFromVPRequestPage/>,
```

Comment:
I rather would have seen that this name is more in line with the other page not using OID4VP, so something like SSICredentialVerifyFromVPRequestPage or something and then rename the other to 'Manual'

Reply:
changed both instances. new values
```
'/information/manual/request': <SSIInformationManualRequestPage/>,
'/credentials/verify/vp/request': <SSICredentialVerifyFromVPRequestPage/>,
```